pytrip/__init__.py

```diff
@@ -24,6 +24,7 @@
 # flake8: noqa
 
 import logging
+from pytrip.cube import Cube
```

Comment:
this means, user may decide fully alone for filename (maybe except for the SVV later, if it becomes relevant and we really want to add it)